ambari-server/conf/unix/log4j.properties

```diff
@@ -79,7 +79,7 @@ log4j.appender.eclipselink.layout=org.apache.log4j.PatternLayout
 log4j.appender.eclipselink.layout.ConversionPattern=%m%n
 
 # Jersey
-log4j.logger.org.glassfish.jersey=WARN,file
+log4j.logger.com.sun.jersey=WARN,file
```

Comment:
You should probably add a new line for `com.sun.jersey` since some of the views use `org.glassfish.jersey`

Reply:
Hi, @rlevas . I have added the `org.glassfish.jersey`  back. Please review it again.